### PR TITLE
Feature/211 statement of case separation

### PR DESCRIPTION
**Pull request by lukaszdrynski**
https://triadmoj.atlassian.net/browse/DIV-211
and 
https://triadmoj.atlassian.net/browse/DIV-212

### CODE REVIEW

File: app/routes/v2.js [hunk 1]
```diff
@@ -268,14 +263,14 @@ var router = express.Router();
...
+  router.get('/reason-for-divorce', function (req, res) {
+    var date = req.session.dateOfWedding_user;
     var separation = 0;
     if (date) {
-      separation = moment.duration(moment().diff(moment(date))).years();
+      separation = moment.duration(moment().diff(moment(date, "DD/MM/YYYY"))).years();
     }
 
-    res.render('v1/reason-for-divorce', {
+    res.render(req.baseUrl.substring(1)+'/reason-for-divorce', {
```
Comment (sophiedrake) [inline app/routes/v2.js]: What does this do?

File: app/routes/v2.js [hunk 2]
```diff
@@ -450,30 +445,45 @@ var router = express.Router();
...
   });
 
-  router.post('/supporting-information/agree', function (req, res) {
-    req.session.supportingInformation.agreeOnEnding = req.body.agreeOnEnding;
-    res.redirect('supporting-information/lived-apart');
+  router.post('/statement-of-case/separation/agree', function (req, res) {
+    var agreeOnEnding=req.body.agreeOnEnding
+    req.session.statementOfCase.agreeOnEnding = agreeOnEnding;
+
+    if (agreeOnEnding==='No'){
```
Comment (sophiedrake) [inline app/routes/v2.js]: add some whitespace?

File: app/routes/v2.js [hunk 3]
```diff
@@ -450,30 +445,45 @@ var router = express.Router();
...
   });
 
-  router.post('/supporting-information/lived-apart', function (req, res) {
+  router.post('/statement-of-case/separation/lived-apart', function (req, res) {
     var livedApart = req.body.livedApart
-    req.session.supportingInformation.livedApart = livedApart;
-    if (livedApart != 'Yes') {
-      res.redirect('supporting-information/exit-time-period-error');
+    req.session.statementOfCase.livedApart = livedApart;
+    if (livedApart == 'No') {
```
Comment (sophiedrake) [inline app/routes/v2.js]: `===`

File: app/views/v2/statement-of-case/separation/lived-apart.html [hunk 4]
```diff
@@ -0,0 +1,34 @@
+{% extends "question.html" %}
+
+{% block question %}
+Have you lived apart for a total of {% if session.facts.reason == 'separation-2-years' %}2{% else %}5{% endif %} years?
```
Comment (sophiedrake) [inline app/views/v2/statement-of-case/separation/lived-apart.html]: You could use the in-line if, I think its something like `{{ "2" if session.facts.reason == 'separation-2-years' else "5" }}`

File: app/routes/v2.js (hunk 1 above)
Comment (lukaszdrynski) [inline app/routes/v2.js]: req.baseUrl gets url passed from server.js

**Pull request merged**